core/controllers/acl_decorators.py

```diff
@@ -2919,6 +2919,23 @@ def test_can_edit(
             raise self.NotFoundException
         if role_services.ACTION_EDIT_ANY_QUESTION in self.user.actions:
             return handler(self, question_id, **kwargs)
+        if role_services.ACTION_EDIT_OWNED_QUESTION in self.user.actions:
```

Comment:
Should this be ACTION_EDIT_QUESTION_IN_MANAGED_TOPIC?

Reply:
Thank you! Changes made.